.pre-commit-config-build.yaml

```diff
@@ -0,0 +1,29 @@
+# this is the set of checks that will be run during build and will
```

Comment:
Please have mercy on my OCD and capitalize the word at the start of a sentence (except if it's a program name)!